src/identity/selectors/index.ts

```diff
@@ -0,0 +1,10 @@
+// Types
+import {AppState} from 'src/types'
+import {CurrentIdentity} from 'src/identity/apis/auth'
+
+export const selectQuartzIdentity = (state: AppState): AppState['identity'] => {
+  return state.identity
+}
+
+export const getQuartzIdentity = (state: AppState): CurrentIdentity =>
```

Comment:
should this be `selectQuartzIdentity`?

Reply:
This selector is an unused dupe, so can be safely deleted.